sources/common/cli_pool.cpp

```diff
@@ -117,3 +117,12 @@ std::optional<std::string> common::Cli::getPassword() const
     }
     return std::nullopt;
 }
+
+uint32_t common::Cli::getApiPort() const
```

Comment:
I forgot to indicate in the CODYNG_STYLE.md that it is necessary to put 2 empty lines between the functions.